microprofile/galleon-common/pom.xml

```diff
@@ -576,6 +582,61 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.jboss.narayana.rts</groupId>
+            <artifactId>lra-service-base</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Please remove this and the other 'exclusions' elements as they are handled in boms/standard-expansion.

A nit but I'm trying to prevent drift in how these handled (and continue to clean up old drift.)